### ISSUE TITLE
[imt] /etc/rcS.d/S02init-arch.sh: local: line 17: not in a function

### ISSUE THREAD

**Issue by vitalivanov** [opencomputeproject/onie#588]
On master:
```
ONIE: OS Install Mode ...
Version   : 2017.08-dirty
Build Date: 2017-08-18T07:05+03:00
Info: Mounting kernel filesystems... done.
/etc/rcS.d/S02init-arch.sh: local: line 17: not in a function
/etc/rcS.d/S02init-arch.sh: local: line 18: not in a function
```
Affecting only Interface Masters HW.

**Comment by vitalivanov** [comment]
Was affecting all Hw before and fixed under bug #452.

**Comment by cbrune** [comment]
Closed by PR #589 and commit bfd2ce0.
